Probesys/bileto
# imp: Improve the whole settings navigation and look

Changes proposed in this pull request:

- Add a "Settings" link in the main header
- Create a global settings page to go to organizations or roles settings
- Add a sidebar on the left of the settings page to navigate quickly
- Rework the page titles to include a breadcrumb as well
- Move the "new organization" anchor in the cards to homogenize with roles

How to test the feature manually:

1. Click on settings
2. Navigate to either "Organizations" or "Roles"
3. Check there is a navigation sidebar on the left
4. Check it works as expected

Pull request checklist:


_If you think one of the item isn’t applicable to the PR, please check it
anyway and precise `N/A` next to it._
